### PR TITLE
Apply pop up on view project description button

### PR DESCRIPTION
**Pull request by byiringiroscar**
#### What does this PR do ❓ 

This PR is raised to implement the following features 🆗 :

- [x] Apply pop-up on the view project description button
- [x] Apply responsiveness in the mobile version

### How should it be tested ⬇️ ⬇️ ⬇️ 

clone this repository
     
```
git clone https://github.com/byiringiroscar/portfolio_desktop_microverse.github.io/
```
Navigate to folder
```
cd portfolio_desktop_microverse.github.io
```
Switch to this branch's view-popup
```
git switch view-popup
```

Start project 😄 

#### Run index.html

Thanks for reviewing my PR 😄

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -78,12 +78,12 @@ <h3 class="mult-intro">Multi-Post Stories</h3>
                     <li><a href="#" class="project-tag-a">Ruby</a></li>
```
Comment (MeqdamAlqudah) [inline index.html:78]: - [x] Please the overlay color should be blur as in the design 

|Yours | Figmas |
| ------------- | ------------- |
| image  | image  |

File: index.html [hunk 2]
```diff
@@ -78,12 +78,12 @@ <h3 class="mult-intro">Multi-Post Stories</h3>
...
```
Comment (MeqdamAlqudah) [inline index.html:79]: - [x] Please the popup should be responsive look how it look in mobile size , the length of this popup is not responsive 
image

File: index.html [hunk 3]
```diff
@@ -106,7 +106,7 @@ <h3>Website Protfolio</h3>
                     <li><a href="#" class="pro-three-a">Ruby</a></li>
                 </ul>
                 <div class="view-project">
-                    <a href="#" id="view-project-id">See Project</a>
+                    <a href="#" id="view-project-id" class="modal-link">See Project</a>
```
Comment (MeqdamAlqudah) [inline index.html:109]: - [x] Please fix the MIT link in the Readme file it is broken 
image
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @byiringiroscar , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

image

 ## Highlights
:+1: no linter errors 
:+1: Javascript

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -1081,6 +1233,45 @@ form button:focus {
...
+  }
+
+  .modal-desc {
+    flex-direction: column;
+    width: 100%;
+    height: 70%;
+    justify-content: space-between;
+    gap: 1px;
+    margin-top: 2px;
+  }
```
Comment (cynthiainga) [inline style.css]: - [ ] Good job so far, but you still have issues to fix with your design. Your design should stick as much as possible to the figma design. As mentioned by the previous reviewer your popup should be responsive also your design should much to the figma design 🙃 🙃 

Your design

Screen Shot 2023-02-09 at 00 04 00

Figma design

Screen Shot 2023-02-09 at 00 05 03

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @byiringiroscar, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Github flow ✔️ 

 ## Required Changes ♻️

- [ ] Please kindly check the comment under the review 👇 👇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Oscar, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**